R/available-data.R

```diff
@@ -1,5 +1,9 @@
 #' Get available data list
 #' 
+#' @description
+#' 
+#' `r lifecycle::badge("stable")`
```

Comment:
Most of the functions are simply stable, is this overkill? It's good to see and relevant packages like `dplyr` in which the API keeps changing.